### ISSUE TITLE
Add Capability to Toggle Plugins/Collections on Swap

### ISSUE THREAD

**Issue by Alaric323** [CallumCarmicheal/XIVControllerToggle#6]
Hi all, I have some of my plugins sorted into collections based on whether or not I am using M&K or Controller.  This plugin is amazing to swap in-game HUD and control scheme, but it is still a manual process to swap plugin collections.  Is it possible to add a "run command on switch" to use Dalamud's native /xltogglecollection command?

Thanks!

**Comment by CallumCarmicheal** [comment]
Not a bad idea, I rarely use collections as most of the plugins I use are QoL. I'll take a look into it and see what can be done.

**Comment by Alaric323** [comment]
Yeah, I have a M&K collection specifically for Mappy, Chat2, and Umbra.  Those three don't work well with controllers so being able to disable them automatically would be amazing!

**Comment by CallumCarmicheal** [comment]
It took some time to get around to finishing the changes, I have implemented the collections and added it to the latest version which should be live soon.

**Comment by Alaric323** [comment]
Awesome!  I'll try it out once the update gets deployed to the Dalamud repo.  Thanks for the work you put in!

**Comment by Alaric323** [comment]
Finally got around to testing things out, works brilliantly!  Thanks for the help!
